tests/wpt/metadata-css/selectors-3_dev/xhtml1/css3-selectors-lang-005.xht.ini

```diff
@@ -1,5 +1,5 @@
 [css3-selectors-lang-005.xht]
   type: testharness
   [A :lang value will match a lang attribute value when the latter contains additional subtags.]
-    expected: FAIL
+    expected: PASS
```

Comment:
By spec, `es` should match `es-MX` so we cannot have a fail here. 
I even checked in browsers and it does work in the same way.